start-goss-servers.sh

```diff
@@ -24,7 +24,12 @@
 #
 # Goss server start up commands to serve health check endpoints
 set -e
-export GOSS_BASE=/opt/cray/tests/install/ncn
+
+if [ "$(hostname)" == "pit" ]; then
```

Comment:
It is safer to look for `/etc/pit-release`

```
redbull-pit:~ # hostname
redbull-pit
redbull-pit:~ # test -f /etc/pit-release && echo pit node
pit node
redbull-pit:~ #
```

Reply:
Sounds reasonable, I'll apply this change in next iteration.

Reply:
Done